packages/ds-medicare-gov/src/components/Dialog/Dialog.jsx

```diff
@@ -3,7 +3,7 @@ import { Dialog, CloseIconThin } from '@cmsgov/design-system';
 
 Dialog.defaultProps = {
   ...Dialog.defaultProps,
-  closeButtonText: null,
+  closeButtonText: '',
```

Comment:
This was a strange artifact I found - the original "close" text for the Dialog close button was still there in Storybook, but transparent so it made the spacing look really strange. This text isn't needed as the button has an aria-label on it, and aria-label's always overwrite the content within.